src/Storefront/Resources/views/storefront/component/product/card/wishlist.html.twig

```diff
@@ -18,7 +18,7 @@
         {% block component_product_wishlist_button %}
             <button
                 class="product-wishlist-{{ productId }} product-wishlist-action{% if appearance == 'circle' %}-circle{% endif %} product-wishlist-not-added product-wishlist-loading"
-                title="{{ "listing.toggleWishlist"|trans|sw_sanitize }}"
```

Comment:
@tinect I'm not sure about this. 🤔 On the detail page it is pretty clear via the "Add to wishlist" text next to the icon. But in the listing we only have the icon by default and there the Tooltip generated by the `title` might be helpful:

![image](https://github.com/shopware/platform/assets/8480203/c22bc4fa-8b9f-4371-b5e2-52ad324cfcc2)

If I investigated correctly, it seems to be valid to have both. The `title` for the tooltip and the `aria-label` for screen-readers. But then it would still be redundant when the text is also there.

Maybe we could alter this as well via `showText` (line 29) and omit or change the aria-label and title when the text is also given. I guess the "coolest" solution would be if the icon labels also change after adding/removing (like the text). Then we could omit those "toggle" or "remove/add" texts and it would always have the correct state.

Reply:
@tobiasberge thanks for the feedback. 

oh 🙈 yes. so changing the entire behaviour would be the best? I'll do so :-)

Reply:
@tobiasberge done 💪

Reply:
@tinect Thanks a lot for the effort. 🎉 We will import the PR.

Reply:
@tinect The storefront wishlist jest tests are now red:

https://github.com/shopware/platform/actions/runs/6411108380/job/17405839953?pr=3339

```
FAIL test/plugin/wishlist/add-to-wishlist.plugin.test.js
  ● AddToWishlistPlugin tests › Add To Wishlist widget plugin exists
    TypeError: Cannot read properties of undefined (reading 'add')
      100 |         this.el.classList.add(this.classList.notAddedState);
      101 |
    > 102 |         this.el.setAttribute('title', this.options.texts.add);
          |                                                          ^
```

The new `texts` options need to be added to the test in order to be accessed.

Reply:
@tobiasberge thank you for your patience. Where and how to add these options? 🤔 I can't find the current once, too.

Reply:
@tinect You can search for the class instantiations (`new AddToWishlistPlugin()`) inside the test `add-to-wishlist.plugin.test.js` and pass the options as the second parameter after the DOM element like this:

```diff
- addToWishlistPlugin = new AddToWishlistPlugin(mockElement);
+ addToWishlistPlugin = new AddToWishlistPlugin(mockElement, {
+     texts: {
+         add: 'Add to wishlist',
+         remove: 'Remove from wishlist' 
+     },
+ });
```

Then the plugin can access the new snippets `this.options.texts.add`. When done everywhere this already makes the test green for me. The coolest would be if we also assert the DOM that the title attributes were applied as we want it.

Alternatively we could also add those new texts as default options with the english texts and they will get overwritten via the template `-options`. Then we don't have to pass them explicitly in the jest test.

Reply:
I've added default text 👍